### ISSUE TITLE
No color output in `git diff`

### ISSUE THREAD

**Issue by xapple** [jupyter/nbdime#474]
My normal `git diff` prints added lines in green and removed lines in red, but as soon as I put `*.ipynb diff=jupyternotebook` in `.gitattributes` I get a colorless output from the same `git diff` command making it not very useful.

**Comment by vidartf** [comment]
First of all, I'm assuming `*.ipynb diff=jupyternotebook` means enabling nbdime in your setup?

What happens if you run `nbdiff` instead of `git diff`? Do you get colors?

**Comment by xapple** [comment]
Yes, by setting that line in the `.gitattributes` within a repository, I enable nbdime in my setup.

I followed your suggestion and ran `nbdiff` directly in the repository, and the colors are present.

**Comment by philippesamuel** [comment]
> First of all, I'm assuming `*.ipynb diff=jupyternotebook` means enabling nbdime in your setup?
> 
> What happens if you run `nbdiff` instead of `git diff`? Do you get colors?

Hi. I have the same issue. When I run ``$ nbdiff``` I still get a colorless output.

**Comment by abergerSRS** [comment]
I too am unable to get color output with either `git diff` or `nbdiff` in a bash terminal window. I have my global config set up with the following:

```
[diff "jupyternotebook"]
    command = git-nbdiffdriver diff --color-words --ignore-outputs --ignore-details`
```

**Comment by vidartf** [comment]
@philippesamuel / @abergerSRS : do you get colors with `git diff` of normal text / code files?

**Comment by abergerSRS** [comment]
@vidartf Yes. I get color output normally

**Comment by vidartf** [comment]
Which OS are you on (actual OS), and what is `os.name` in your python ?

**Comment by abergerSRS** [comment]
I'm running Windows 10, and `os.name` returns `nt`

**Comment by TimAdamson21** [comment]
I'm having the same issue on Windows 10 as well.
